tests/tracking/_model_registry/test_utils.py

```diff
@@ -102,6 +102,21 @@ def test_get_store_bad_uris(bad_uri):
         _get_store()
 
 
+def test_get_store_caches_on_store_uri():
+    store_uri_1 = "sqlite:///store1.db"
+    store_uri_2 = "sqlite:///store2.db"
```

Comment:
Can we create these database files in a temporary directory?

Reply:
Done! Also applied this suggestion to other test cases added by this PR.